Exec/science/flame/analysis/profiles.py

```diff
@@ -12,6 +13,28 @@
 import matplotlib.pyplot as plt
 
 import yt
+yt.set_log_level(50)
+
+dt = 0.005
+tmax = 0.05
+
+times = np.arange(0.0, tmax + dt, dt)
+
+def find_files(plist):
+
+    mask = np.zeros(len(times))
+    files_to_plot = []
+    for pfile in plist:
+        ds = yt.load(pfile, hint="castro")
+        for k, t in enumerate(times):
+            if mask[k]:
+                continue
+            if ds.current_time >= t:
+                files_to_plot.append(pfile)
+                print(f"plotting {pfile} at t = {t}")
+                mask[k] = 1.0
```

Comment:
We should break out of the `times` loop here, so the same file doesn't get plotted multiple times (if the plotfiles are spaced out longer than `dt`). Looks good otherwise.

Reply:
yes, I think you are right.  I'll fix that.

Reply:
updated